src/mount/sauna_client.cc

```diff
@@ -233,6 +233,7 @@ uint8_t get_session_flags() { return session_flags; }
 bool isMasterDisconnected() { return gIsDisconnectedFromMaster.load(); }
 
 #define LOCAL_USERS_THRESHOLD 0x30000
+#define IIS_USER 17
```

Comment:
I suggest to use a more generic name, like `ALLOWED_USERS` to use this feature not only for IIS in Tsukaeru, but for other customers who could have the same issue.

I agree with @aNeutrino that if we use a list of users, is even more flexible and generic the solution.
A definition like
`std::unordered_set<int> ALLOWED_USERS;`
can do the job.